Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1313,6 +1315,17 @@ void AXObjectCache::onEventListenerRemoved(Node& node, const AtomString& eventTy
 #endif // ENABLE(ACCESSIBILITY_ISOLATED_TREE)
 }
 
+void AXObjectCache::onExpandedChanged(HTMLDetailsElement& detailsElement)
+{
+    postNotification(get(detailsElement), AXExpandedChanged);
```

Comment:
do we want to post this after ITM is scheduled to do work? It seems possible that there could be a gap where the notification is posted, but ITM is scheduled to update in time

Reply:
When notifications are processed as a result of `postNotification`, the isolated tree is updated at the same time (and actually, before the notification goes out to ATs):

```cpp
void AXObjectCache::notificationPostTimerFired()
{
    ...truncated...

    auto notifications = std::exchange(m_notificationsToPost, { });

    ...truncated...

    updateIsolatedTree(notificationsToPost);

    for (const auto& note : notificationsToPost)
        postPlatformNotification(note.first, note.second);
}
```

So I think it should be OK, but let me know if I'm not understanding you correctly.